prime-router/src/main/kotlin/cli/ProcessDataCommands.kt

```diff
@@ -483,6 +483,7 @@ class ProcessData(
                             "emptyReceiver",
                             "emptyOrganization",
                             "covid-19",
+                            "unknown",
```

Comment:
Instead of "unknown", probably can just use the enum's `inactive` value.

Reply:
Updated